README.md

```diff
@@ -0,0 +1,46 @@
+Project Name
+Is-haq Adnan's Personal Portfolio
+
+Built With
+Html
+Css
+linter
+node
+
+Getting Started
+This is a development about my developments and expertise
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Basic knowledge of html, css, and linters
+
+Setup
+Visual studio code
+
+Install
+node package manager
+
+Authors
+👤 Author1
+GitHub: @AdnanOlarmmi
+Twitter: @AdnanIs71755510
+LinkedIn: Adnan (Olamilekan) Is-haq  (www.linkedin.com/in/adnan-is-haq-olamilekan)
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
- [ ] [OPTIONAL] Kindly consider using the same markdown for the README as the microverse one, so it looks professional and more appealing 

- Your README
![Screenshot_20220818_180804](https://user-images.githubusercontent.com/25227022/185454135-998085e7-4295-417d-bdbd-3b3be57518db.png)

- Microverse README (with markdown)
![Screenshot_20220818_180856](https://user-images.githubusercontent.com/25227022/185454310-09e718fb-327b-4b84-ad0e-5febcf30eec1.png)
![Screenshot_20220818_180909](https://user-images.githubusercontent.com/25227022/185454330-d2d8e081-f01b-4ba4-8e43-44018276b63a.png)

Reply:
@Ismailco Hi, please where can I get that template for readme file to copy into vscode, directly. so I can edit it to my projects, thank you. Can you please share the file or a link to the file? Tried copying from github, but it's still the same thing as that.